docs/docs/rest-api/public/api/v2/types/dropwizardMetrics.raml

```diff
@@ -0,0 +1,143 @@
+#%RAML 1.0 Library
+types:
+  DropwizardCounter:
```

Comment:
dislike exposing our implementation details "dropwizard" to the outside world

Reply:
based on maintaining both kamon and drop... perhaps we have to :(

Reply:
Probably I am missing something here. In which way it is exposed to the outside world? I can surely drop `Dropwizard` prefix, and replace `Kamon` with `Deprecated`.